cli/dcoscli/node/main.py

```diff
@@ -0,0 +1,92 @@
+"""Manage DCOS nodes
+
+Usage:
+    dcos node --info
+    dcos node [--json]
+
+Options:
+    -h, --help    Show this screen
+    --info        Show a short description of this subcommand
+    --json        Print json-formatted nodes
+    --version     Show version
+"""
+
+import dcoscli
+import docopt
+from dcos import cmds, emitting, errors, mesos, util
+from dcos.errors import DCOSException
+from dcoscli import tables
+
+logger = util.get_logger(__name__)
+emitter = emitting.FlatEmitter()
+
+
+def main():
+    try:
+        return _main()
+    except DCOSException as e:
+        emitter.publish(e)
+        return 1
+
+
+def _main():
```

Comment:
Out of scope for this PR but should we refactor this dance (`main` and `_main`) and move it to `dcos`? I haven't thought it much but it feels useful for other sub-commands.